corehq/apps/userreports/tasks.py

```diff
@@ -54,7 +54,7 @@ def _build_indicators(indicator_config_id, relevant_ids):
         redis_client.delete(redis_key)
 
 
-@task(queue='ucr_queue', ignore_result=True, acks_late=True)
+@task(queue='ucr_queue', ignore_result=True)
 def rebuild_indicators(indicator_config_id):
```

Comment:
I haven't been following all the changes to data source rebuilding super closely, but I thought it was ok to retry this task?
If it's not ok to retry it, then would it be an issue if someone clicked the "Rebuild Data Source" button a second time before the task has finished?